src/App.js

```diff
@@ -1,11 +1,22 @@
 import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Quote from './components/Quote';
 import Calculator from './components/Calculator';
+import Footer from './components/Footer';
 
 function App() {
   return (
     <div className="App">
-      <h2>Let&apos;s do some Calculations!</h2>
-      <Calculator />
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Quote />} />
```

Comment:
- [ ] Kindly render the `Quote` component into your main component `App` next to the `Calculator` as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=It%20should%20be%20on%20the%20same%20page%20as%20the%20Calculator) instead of creating a new route for the `Quote`.